### PR TITLE
Change Geometry_cff to GeometryDB_cff in AlCaDB related config files

### PR DESCRIPTION
**Pull request by tvami**
#### PR description:

Taking care of the AlCa and DB part of https://github.com/cms-sw/cmssw/issues/31113
i.e. the fact that 
`process.load("Configuration.StandardSequences.Geometry_cff")`
was removed a while ago: https://github.com/cms-sw/cmssw/pull/8810
and thus should be replaced with 
`process.load("Configuration.StandardSequences.GeometryDB_cff")`

#### PR validation:

`scramv1 b runtests` though I guess this doesn't test any of these files, otherwise it would have been fixed already

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Not a backport and no backport is needed

### CODE REVIEW

**Comment by tvami** [comment]
@cmsbuild , code checks

**Comment by tvami** [comment]
@cmsbuild , please test

File: Alignment/MuonAlignment/test/make_SVGtemplates_cfg.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 process.source = cms.Source("EmptySource")
 process.maxEvents = cms.untracked.PSet(input = cms.untracked.int32(1))
 
-process.load("Configuration.StandardSequences.Geometry_cff")
+process.load("Configuration.StandardSequences.GeometryDB_cff")
```
Comment (mmusich) [inline Alignment/MuonAlignment/test/make_SVGtemplates_cfg.py:7]: If I am not mistaken a Global Tag (or at least an ESSource providing the geometry records in the ES) is needed in order for `GeometryDB_cff` to work. 
As far as I can see, that is not included in this file (and in many others of this PR).

**Comment by malbouis** [comment]
+alca

Even though there seems to be missing GTs in some of the files, as @mmusich pointed out, we think this can move on as is and AlCa will be contacting the subsystems for a more a detailed check on these files and possible removal in case they are not used anymore.

**Comment by malbouis** [comment]
+db

**Comment by mmusich** [comment]
> we think this can move on as is 

seems you are trading a broken configuration for a differently broken one, not sure to see the advantage, also considering that some of these might have been explicitly designed in order to take the geometry from XML.

> and AlCa will be contacting the subsystems for a more a detailed check on these files and possible removal in case they are not used anymore.

that seems indeed the way to go.
My 2 cents.

**Comment by malbouis** [comment]
> > we think this can move on as is
> 
> seems you are trading a broken configuration for a differently broken one, not sure to see the advantage, also considering that some of these might have been explicitly designed in order to take the geometry from XML.
> 

I see what you mean. I think there will be still a slight improvement, since some of the files that were changed might work again. The other reason for the change is that the original cff config file does not exist anymore, so it was accessing neither the DB nor XML. 

I agree though it is not optimal, but still see some improvement and a reason for going forward with this now.

> > and AlCa will be contacting the subsystems for a more a detailed check on these files and possible removal in case they are not used anymore.
> 
> that seems indeed the way to go.
> My 2 cents.

Thanks! We will follow up with that!

**Comment by perrotta** [comment]
> 
> Even though there seems to be missing GTs in some of the files, as @mmusich pointed out, we think this can move on as is and AlCa will be contacting the subsystems for a more a detailed check on these files and possible removal in case they are not used anymore.

Given what you write, I have the impression that all these scripts are not used by now.
Instead of investing time in trying to fix them, shouldn't be more convenient to just remove them? Once fixed by adjusting the name of Geometry_cff, it will be more difficult to identify them as outdated.

**Comment by tvami** [comment]
> > Even though there seems to be missing GTs in some of the files, as @mmusich pointed out, we think this can move on as is and AlCa will be contacting the subsystems for a more a detailed check on these files and possible removal in case they are not used anymore.
> 
> Given what you write, I have the impression that all these scripts are not used by now.
> Instead of investing time in trying to fix them, shouldn't be more convenient to just remove them? Once fixed by adjusting the name of Geometry_cff, it will be more difficult to identify them as outdated.

@perrotta I think our idea was to email the subsystems to have a look at these files based on the file change list in this PR

**Comment by perrotta** [comment]
> @perrotta I think our idea was to email the subsystems to have a look at these files based on the file change list in this PR
> I should also add that it should be the subsystem's decision to delete these files (which are indeed not used, since the original `Geometry_cff` doesn't exist anymore)

Sure!
I suggest to email the subsystems before merging this PR, suggesting them that the very first fix they must apply in case they want to resurrect these codes is what implemented here. Alternatively, they could delete them. 
By letting the wrong include at its place for now would allow identifying more easily in the future which files haven't been updated: one could then even decide to simply forcedly  remove them.

**Comment by perrotta** [comment]
+1
- Given the message sent by @tvami to the developers of thes code, which asks for either remove the files, if they are not used, or modify them (by adding the appropriate GT) on top of the first part of the fix implemented here, let merge this PR as is
- To be considered an intermediate step

**Pull request merged**